CHANGELOG.md

```diff
@@ -73,6 +73,9 @@ constructor. ([#6240](https://github.com/biocore/scikit-bio/issues/624))
 * ``DissimilarityMatrix.plot()`` no longer leaves a white border around the
   heatmap it plots (PR #1070).
 
+### Deprecated functionality [stable]
+* `skbio.Sequence.copy` has been deprecated in favor of `copy.copy(seq)` and `copy.deepcopy(seq)`.
```

Comment:
:+1:, Especially with our current docs with explicitly point out the existence of `copy.copy` and `copy.deepcopy`